primitives/src/v1.rs

```diff
@@ -721,11 +735,12 @@ pub enum CandidateEvent<H = Hash> {
 
 /// Information about validator sets of a session.
 #[derive(Clone, Encode, Decode, RuntimeDebug)]
-#[cfg_attr(feature = "std", derive(PartialEq, Default))]
+#[cfg_attr(feature = "std", derive(PartialEq, Default, MallocSizeOf))]
 pub struct SessionInfo {
 	/// Validators in canonical ordering.
 	pub validators: Vec<ValidatorId>,
 	/// Validators' authority discovery keys for the session in canonical ordering.
+	#[cfg_attr(feature = "std", ignore_malloc_size_of = "outside type")]
```

Comment:
:disappointed: we should really implement https://github.com/paritytech/parity-common/issues/255